docs/ros-snap.md

```diff
@@ -349,15 +349,13 @@ you'll have a .snap.
 
 ### Take the .snap for a test drive
 
-You can transfer your newly-minted .snap to your Ubuntu Core machine and install
-it at the same time via `snappy-remote`, for example:
+Transfer your newly-minted .snap to your Ubuntu Core machine with `scp`,
+then install it:
 
```

Comment:
Or should we? Should we just assume people are running snaps on the desktop now?